python/smashbox/__init__.py

```diff
@@ -0,0 +1 @@
+from . import webdav as webdav
```

Comment:
You should not need the `as webdav` part.

Reply:
It isn't needed syntactically, but running [mypy in strict mode](https://github.com/aio-libs/aiohttp/issues/3868) will complain about it. Although I can remove it because we haven't really decided on whether we are going to enforce a type-checker in the first place.